editor/test/integration/extension_spine_test.clj

```diff
@@ -31,7 +31,9 @@
 
 (set! *warn-on-reflection* true)
 
-(defonce ^:private extension-spine-url "https://github.com/defold/extension-spine/archive/main.zip")
+;; HACK: Don't forget to restore the url once we've merged the change into extension-spine.
+;; (defonce ^:private extension-spine-url "https://github.com/defold/extension-spine/archive/main.zip")
+(defonce ^:private extension-spine-url "https://github.com/defold/extension-spine/archive/refs/heads/editor-node-msg-avoid-texture-size.zip")
```

Comment:
Once we've merged https://github.com/defold/extension-spine/pull/113, we can change this back to use the `main` release instead.